content/en/universal_service_monitoring/_index.md

```diff
@@ -46,10 +47,10 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 
 ### Prerequisites
```

Comment:
Could we add "containers" (Linux) as a prereq in this section? We've seen some customers try setting this up on non-container Linux setups

Reply:
In line 53 we talk about non-container setups. Are they supported (i.e., do you just mean to indicate we don't support serverless) or should we remove line 53 and claim support only for containerized linux services?

Reply:
I think we should remove line 53 to avoid this confusion altogether. We don't officially support non-container setups yet (e.g. baremetal on a VM or serverless)

Reply:
So we should make it clear, potentially under `Supported Platforms` that containers are required